project_event/report/project_task_activity_report.py

```diff
@@ -78,17 +79,13 @@ def get_tasks_details(self, tasks):
                     'resource': resources_list,
                 })
             elif task.resource_type == 'equipment' and task.equipment_id:
```

Comment:
You can merge the two if statements. And is task.resource_type == 'equipment' the same as task..equipment_id.category_type == 'equipment'?

Reply:
I combined all the if condition to remove nested code